projects/addon-table/components/table/directives/direction-by-order.directive.ts

```diff
@@ -0,0 +1,23 @@
+import {Directive, Inject, Input, Output} from '@angular/core';
+import {map} from 'rxjs/operators';
+
+import {TuiTableDirective} from './table.directive';
+
+@Directive({
+    selector: `table[tuiTable][tuiDirectionByOrder]`,
+})
+export class TuiDirectionByOrderDirective<T> {
```

Comment:
What do you think, will you be able to add a unit test or a demo example?

Reply:
yes, will do both :) will update soon